web/js/map-OpenLayers.js

```diff
@@ -925,7 +935,21 @@ OpenLayers.Control.PermalinkFMS = OpenLayers.Class(OpenLayers.Control.Permalink,
         if ( alter_zoom ) {
             zoom += fixmystreet.zoomOffset;
         }
-        href += separator + OpenLayers.Util.getParameterString(this.createParams(center, zoom));
+
+        var params = this.createParams(center, zoom);
+        // Roundtripping a param containing urlencoded commas through
+        // createParams and getParameterString results in the commas
+        // becoming un-urlencoded, so let's avoid this by joining the arrays
+        // with commas here, which will then be urlencoded correctly by
+        // getParameterString.
+        if (OpenLayers.Util.isArray(params.status)) {
+            params.status = params.status.join(",");
+        }
+        if (OpenLayers.Util.isArray(params.filter_category)) {
+            params.filter_category = params.filter_category.join(",");
+        }
+
+        href += separator + OpenLayers.Util.getParameterString(params);
```

Comment:
Now I notice this puts an awful lot of layers in the URL (due to all the hidden road ones) - should we strip that out?

Reply:
Should it remove the `pc` parameter if there's a lat/lon, assuming those take precedence.

Reply:
Good idea - although will removing `pc` prevent the "user originally entered" text being captured if they make a report from that page?

Reply:
It's also in a hidden field on the page, so I think it'll still get remembered through the form submission.